response_operations_ui/views/messages.py

```diff
@@ -330,3 +334,10 @@ def _get_human_readable_date(sent_date):
 
 def _get_unread_status(message):
     return 'UNREAD' in message.get('labels', [])
+
+
+@messages_bp.after_request
+def disabling_caching_headers(response):
```

Comment:
This method name is a little misleading, as it could be read as disabling the cache headers i.e. not setting them. I think it would be clearer as `disable_caching`